sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -36,8 +40,16 @@ public static class TestDataUtilities
         public const string LusidCashIdentifier = "Instrument/default/Currency";
         public const string LusidInstrumentIdentifier = "Instrument/default/LusidInstrumentId";
 
-        public static DateTimeOffset EffectiveAt = new DateTimeOffset(2020, 1, 2, 0, 0, 0, TimeSpan.Zero);
-
+        public static DateTimeOffset StartDate = new DateTimeOffset(2020, 1, 2, 0, 0, 0, TimeSpan.Zero);
+        public static DateTimeOffset EffectiveAt = StartDate;
+        public static DateTimeOffset ResetDate = StartDate.AddYears(-1); // Temp reset date -> will change examples to pull resets from instruments when implemented
+        
+        public static List<decimal> exampleRates1 = new List<decimal> { 1.0m, 0.992548449440757m, 0.985152424487251m, 0.977731146620901m, 0.970365774179742m, 0.85m };
```

Comment:
(variable naming) What rates are these, discount curve rates?

Reply:
Agreed not clear - will update.